assets/volcanoplot-style.css

```
@@ -1,106 +1,142 @@
 #vp-page-content {
+    font-family: 'Open Sans Light', sans-serif;
+    background-color: #F1F9FA;
+    color: #5B5B5B;
     width: 100%;
-    display: flex;
-    flex-direction: row;
-    align-items: center;
-    align-content: space-evenly;
+    min-height: calc(100vh - 60px);
 }
 
-#vp-info-panel-div {
-    width: 70%;
-    margin: 15px;
-    border: solid 2px #C8D4E3;
-    border-radius: 5px;
-    padding: 10px;
+#vp-page-content ::-webkit-scrollbar {
+    width: 4px;
+    height: 0px;
+    margin-right: 500px;
+    background-color: #F1F9FA;
 }
-
-#vp-controls-div {
-    margin-top: 20px;
-    display: flex;
-    flex-direction: row;
-    align-items: center;
-    justify-content: space-evenly;
+#vp-page-content ::-webkit-scrollbar-thumb {
+    width: 2px;
+    background-color: #19D3F3;
+    border-radius:15px;
+    padding-right:5px;
 }
 
-#vp-indicators-div {
-    margin-top: 30px;
-    display: flex;
-    flex-direction: row;
-    align-items: center;
-    justify-content: space-evenly;
+#vp-page-content::-webkit-scrollbar {
+    display: none;
 }
 
-#vp-graph-div{
-    width: 100%;
-    align-items: center;
+#vp-control-tabs {
+    width: 400px;
+    height: 550px;
+    margin: 50px;
+    margin-right: 0px;
+    background-color: #fff;
+    font-size: 10pt;
+    float: left;
 }
 
-#vp-graph {
-    height: 600px;
-    margin-top: 2px;
-}
 
-#vp-dataset-dropdown{
-    width: 140px;
+#vp-control-tabs .tab-content {
+    padding: 20px;
 }
 
-.vp-vertical-style {
-    display: flex;
-    flex-direction: column;
-    align-items: center;
-    justify-content: space-between;
+.vp-tab {
+    height: 440px;
+    overflow-y: auto;
+    padding-right: 20px;
+    margin: 20px;
 }
 
-.vp-horizontal-style {
-    display: flex;
-    flex-direction: row;
-    align-items: center;
-    justify-content: space-between;
+#vp-control-tabs .tab {
+    background-color: #fff;
```

```suggestion
    background-color: #FFFFFF;
```
for sake of notation uniformity